docs/tutorials/plans-crowdloan-datahighway-kusama-tanganika.md

```diff
@@ -0,0 +1,226 @@
+---
+id: plans-crowdloan-datahighway-kusama-tanganika.md
+title: Crowdloan Plan for Tanganika Network (DataHighway's Kusama Parachain)
+sidebar_label: Crowdloan Plan for Tanganika Network (DataHighway's Kusama Parachain)
+---
+
+## About the Tanganika Network Crowdloan?
+
+To launch the Tanganika Network on Kusama we need help to bootstrap it. The project must secure enough funds to cover the cost of renting ("leasing") a parachain slot on the network. To meet this objective, the DataHighway team will run a crowdloan campaign on behalf of the network. For the Tanganika crowdloan, individuals agree to lock up a portion of their KSM in exchange for the DHX native network token. These tokens are locked from the point of participation until the slot is successfully won, and then through the entirety of the lease period (48 Weeks). A pool of **300,000 (300k) DHX** tokens representing **1%** of the DataHighway Treasury's Unlocked Reserves comprising 30,000,000 DHX tokens, which equates to **0.3%** of the DataHighway's network's genesis total supply will be distributed to crowdloan participants as a reward (proportionate to their share of the overall KSMs contributed to the crowdloan) throughout the lease period.
+The DHX tokens shall be bridged between the DataHighway's standalone chain which will stop to operate, and its Tanganika parathread/parachain network on the Kusama. 
+
+
+## Rewards
+
+### Token Reward Pool
+
+**300k DHX** tokens (**0.3%** of the DataHighway's genesis token supply).
+
+### Contributor Rewards
+
+Paid as a % of your contribution to the pool (your KSM Contribution / Total KSM Contributed) * **300k DHX** tokens.
+
+### Reward Distributions
+
+**30%** of the reward pool (**90k DHX** tokens) will be distributed within a reasonable timeframe upon successful launch and transfers being enabled. The remaining **70%** of the reward pool (**210k** DHX tokens) will vest over the slot lease duration (48 weeks).
+
+## Timing
+
+### Crowdloan Start Date
+
+If a crowdloan is used then the first one commencing in April 2022 will be targeted.
+
+### Lease Duration
+
+We wish to use the crowdloan to secure a slot as a Kusama parachain for the maximum duration of 48 weeks.
+
+### Token Lock Period
+
+KSM tokens contributed to the crowdloan will be locked from the time they are contributed until the end of the parachain lease.
+
+## Participation Requirements
+
+### Tokens Needed
+
+KSM. If you do not yet have KSM, you will need to purchase some tokens in order to participate. If you already have KSM, ensure that it is located in a non-custodial wallet like Polkadot.js, and does not have any staking or democracy holds.
+
+You need KSM stored either in a non-custodial (non-exchange) address, or through a partner exchange (to be announced).
```

Comment:
Will there be a dApp integrated in Datahighway.com for contributing? Or can we use Bifrost and/or Parallel?

Reply:
> Will there be a dApp integrated in Datahighway.com for contributing?
no, not unless otherwise advised

> Or can we use Bifrost and/or Parallel?
no, not unless otherwise advised